src/libstore/store-api.hh

```diff
@@ -20,12 +20,12 @@
 namespace nix {
 
 
-MakeError(SubstError, Error)
-MakeError(BuildError, Error) /* denotes a permanent build failure */
-MakeError(InvalidPath, Error)
-MakeError(Unsupported, Error)
-MakeError(SubstituteGone, Error)
-MakeError(SubstituterDisabled, Error)
+MakeError(SubstError, Error);
+MakeError(BuildError, Error) /* denotes a permanent build failure */;
```

Comment:
This technically works, but it'd be nicer to put the `;` next to the actual expression instead of after the comment.

Reply:
Hah! Too much sed